src/view/subgraph.rs

```diff
@@ -321,6 +310,80 @@ where
     }
 }
 
+/// An error from [Subgraph::copy_in_parent]
+#[derive(Clone, Debug, PartialEq, Eq, Error)]
+pub enum CopySubgraphError {
+    /// Tried to copy an edge crossing a subgraph boundary
+    /// when the containing graph is not a [MultiMut](crate::MultiMut)
+    #[error("Cannot copy edge between external port {external:?} and (copy of) internal port {internal:?}")]
+    CantCopyBoundary {
+        internal: PortIndex,
+        external: PortIndex,
+    },
+}
+
+impl<G: LinkMut> Subgraph<G>
+where
+    G::LinkEndpoint: Into<PortIndex>,
+{
+    /// Copies all the nodes and edges in this subgraph into the parent graph.
```

Comment:
The first line is used as a short description on module listings.
```suggestion
    /// Copies all the nodes and edges in this subgraph into the parent graph.
    ///
```